docs/shared/nx-core.md

```diff
@@ -47,9 +53,15 @@ package.json
 {
   "extends": "@nrwl/workspace/presets/core.json",
   "npmScope": "myorg",
+  "affected": {
+    "defaultBase": "main"
+  },
+  "cli": {
+    "defaultCollection": "@nrwl/workspace"
+  },
```

Comment:
wasn't sure if we should include this or not but added it to be more true to what is created